samples/world-clock/README.md

```diff
@@ -0,0 +1,69 @@
+World Clock
+===========
+
+This mobile app sample has these modules:
+
+ * **presenters** is a Kotlin/Multiplatform library that shows the time in various time zones.
+ * **android** and **iOS** apps continuously fetch the latest presenter and run it.
+
+Prerequisites
+-------------
+
+In order to build and run these applications you'll need to:
+
+ * Have Android Studio installed
+ * Have `cmake` installed (on Mac, you can use `brew install cmake`)
+ * Build QuickJS for your platform using `./.github/workflows/build-mac.sh` (replace `-mac` with the
```

Comment:
The script autodetects locally and only needs override parameters for CI

Reply:
Yes! Though I think we do need `-mac` or `-linux`.